### PR TITLE
feat: show hide password in signup

### PR DESCRIPTION
**Pull request by sexta13**
### What does this do?

- Shows/hides the password field. 
- Creates a new component PasswordInput

### CODE REVIEW

File: src/components/PasswordInput/PasswordInput.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import React, { useState } from "react";
+
+const styles = theme => ({
+  eye: {
+    cursor: "pointer"
+  }
+});
+
+const PasswordInput = props => {
+  const [passwordIsMasked, setPasswordIsMasked] = useState(true);
```
Comment (luispedro18) [inline src/components/PasswordInput/PasswordInput.js:16]: just a matter of preference.. I usually start my boolean variables with `is` e.g ìsPasswordMasked` in this case..

Comment (sexta13) [reply src/components/PasswordInput/PasswordInput.js:16]: me too...well spotted. I was following a tutorial about material ui and I just missed that. I'll change it

File: src/components/PasswordInput/PasswordInput.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <TextField
+      type={passwordIsMasked ? "password" : "text"}
+      {...props}
+      InputProps={{
+        endAdornment: (
+          <InputAdornment position="end">
+            <RemoveRedEye
+              className={classes.eye}
+              onClick={togglePasswordMask}
+            />
```
Comment (luispedro18) [inline src/components/PasswordInput/PasswordInput.js:33]: Maybe use these 2 icons and toggle them according with passwordIsMasked value?
Screenshot 2020-04-15 at 17 54 26

File: src/components/PasswordInput/PasswordInput.js (hunk 1 above)
Comment (luispedro18) [inline src/components/PasswordInput/PasswordInput.js:16]: just a matter of preference.. I usually start my boolean variables with is e.g ìsPasswordMasked` in this case..

Comment (sexta13) [reply src/components/PasswordInput/PasswordInput.js]: changed it.

File: src/components/PasswordInput/PasswordInput.js (hunk 2 above)
Comment (sexta13) [reply src/components/PasswordInput/PasswordInput.js]: nice. :) just did the change

**Pull request merged**